packages/acceptit-Core.package/ACUserStory.class/class/buildArgumentsForMethod.using..st

```diff
@@ -5,5 +5,5 @@ buildArgumentsForMethod: aMethod using: aString
 	selector :=  (aMethod pragmas detect: [:pragma | pragma keyword = #step:]) arguments first.
 	^ (ACTranslator 
 		bindParametersInSelector: selector 
-		toValuesIn: aString withoutLeadingSubject withLeadingAuxiliaryNormalized)
```

Comment:
I will have to check this change tomorrow again, because I originally assumed that the error has to be in ACTranslaor bindParametersInCollection: aCollection toValuesIn: aString

Reply:
You can check by debugging what get's parsed into that method and why the results are so weird. It was just the case that the subject ("I") was being removed so the steps didn't match anymore and some indices which were used to get the variables out of the step call where shifted and therefore wrong.

I think the intent of `withoutLeadingSubject` ist that the actor doesn't really matter for the steps content.
`i expect A to be xyz` should be the same as `We expect A to be xyz` or `Someone expects A to be xyz` The code inside of the step should always be the same so having multiple steps for it is kind of a waste. But at the same time we can't know all the actors that a project might have and that might be mentioned in the steps. So we can't neutralized the steps completely anyways. Except if we always just remove the first word but that does seem a little overkill and I have never read that cucumber does it this way.

So I think just fixing it this way should be fine.